docs/source/getting-started/intro-quickstart/index.rst

```diff
@@ -0,0 +1,229 @@
+MLflow Tracking Quickstart
+==========================
+
+Welcome to MLflow! 
+
+The purpose of this quickstart is to provide a quick, no-nonsense, bare-bones guide to the most essential core APIs of MLflow Tracking. 
```

Comment:
```suggestion
The purpose of this quickstart is to provide a quick guide to the most essential core APIs of MLflow Tracking. 
```

Reply:
minor nit: might keep it formal, I like the adjectives though.

Reply:
removed